clang-tools-extra/test/clang-doc/single-source-html.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
```

Comment:
```suggestion
// RUN: rm -rf %t && mkdir %t
```

Reply:
These still seem to be different steps. combining them is intentional. Actually, do you even need this directory? I don't think you do.